internal/deviations/deviations.go

```diff
@@ -675,4 +680,6 @@ var (
 	_ = flag.Bool("deviation_interface_ref_config_unsupported", false, "Device does not support interface-ref configuration when applying features to interface")
 
 	_ = flag.Bool("deviation_storage_component_unsupported", false, "Set to true for device that does not support telemetry path /components/component/storage")
+
+	gnoiFabricComponentRebootUnsupported = flag.Bool("deviation_gnoi_fabric_component_reboot_unsupported", false, "Set true if the device does not support using gNOI to reboot the Fabric Component.")
```

Comment:
No need to add new flags. Add it to the metadata proto instead

Reply:
Thanks for your comment, Greg. The comment has been implemented.